src/Common/perf/PerfRunner/PerfRunner.csproj

```diff
@@ -1,6 +1,4 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
@eerhardt are you aware of any way we can avoid setting this string in all our projects? We are very likely going to need to move to another SDK as part of our arcade repo and it would be great if we could just change it in one place in the repo if possible.

Reply:
Based on https://github.com/Microsoft/msbuild/issues/1493, we can use `<Import>` semantics, because in `Project Sdk="$(prop)"`,  `prop` does not be evaluated, nor in `<Sdk Name="$(prop)">`. Alternatively, we can implement `ISdkResolver` when it is implemented in MSBuild. :)

Reply:
The error above indicates that you are not importing Microsoft.Common.targets and indirectly Microsoft.Managed.DesignTime.targets. You can verify this by doing a `msbuild /pp` on the project.

Reply:
You can import the SDK in a central props/targets, but then the central props/targets can't be `Directory.Build.*`. Chicken and egg. Sdk imports common targets and common targets imports `Directory.Build.*`

If you use explicit dir.props/dir.targets, then you can do this: nguerrera/sdkimportrepro@6d34578

Reply:
Oh I see that @eerhardt  explicitly imported `Directory.Build.*` and then turned off the magic import. That should have worked too. My above commit with different name for explicit directory props/targets is working fine in VS. Let me try @eerhardt's approach in that sandbox.

Reply:
https://github.com/nguerrera/sdkimportrepro/commit/ab90e6218bc67b55d4e3aa311881261849ea5cfb works fine in VS for me too. I cannot repro what @eerhardt is seeing with that approach.

Reply:
I tried it on today's daily dogfood build, and am no longer seeing the error.  The project loads and builds correctly in VS.